File: 3/03226.cpp
#include <iostream>
using namespace std;

int H, M;

void plusMin() {
	M++;
	if (M == 60) { H++; M = 0; }
	if (H == 24) H = 0;
}

int main() {
	int N; cin >> N;

	int price = 0;
	for (int j = 0; j < N; j++) {
		int D; char colon;
		cin >> H >> colon >> M >> D;

		for (int i = 0; i < D; i++) {
			if (H >= 7 && H < 19) price += 10;
			else price += 5;

			plusMin();
		}
	}

	cout << price << endl;
}

File: 3/03226.py
global H, M

def plusMin():
    global H, M

    M += 1
    if M == 60:
        H += 1
        M = 0
    if H == 24:
        H = 0

price = 0
for i in range(int(input())):
    time, D = input().split()
    H, M = map(int, time.split(':'))
    D = int(D)

    for j in range(D):
        if H >= 7 and H < 19: price += 10
        else: price += 5

        plusMin()

print(price)
